docusaurus.config.js

```diff
@@ -5,7 +5,22 @@ const lightCodeTheme = require("prism-react-renderer/themes/github");
 const darkCodeTheme = require("prism-react-renderer/themes/dracula");
 const versionsArchived = require("./versionsArchived.json");
 
+
+
+// Precompute the copyright for each locale
+const currentYear = new Date().getFullYear();
+const localizedCopyrights = {
+  en: `${currentYear} © KubeEdge Project Authors. All rights reserved.`,
+  zh: `${currentYear} © KubeEdge 项目作者。版权所有。`,
```

Comment:
Do we need to have this Chinese version of footer in here? i think this is meant to be English version only, so i think it works okay with current implementation.